ChopChop/Views/Recipes/RecipeCollectionView.swift

```diff
@@ -14,6 +14,9 @@ struct RecipeCollectionView: View {
         VStack {
             SearchBar(text: $viewModel.query, placeholder: "Search recipes...")
             HStack {
+                NavigationLink(destination: RecipeFormView(viewModel: RecipeFormViewModel())) {
+                    Text("Add recipe")
```

Comment:
Alternatively you can use `Image(systemName: "plus")` for the label of the nav link to be consistent w the rest of the app